sources/commands/Base.ts

```diff
@@ -16,10 +16,10 @@ export abstract class BaseCommand extends Command<Context> {
           throw new UsageError(`Couldn't find a project in the local directory - please explicit the package manager to pack, or run this command from a valid project`);
```

Comment:
```suggestion
          throw new UsageError(`Couldn't find a project in the local directory - please specify the package manager to pack, or run this command from a valid project`);
```

Reply:
Would you like to send a separate PR? That feels a bit unrelated

Reply:
Sure. I hadn't really checked where this came from.